backend/src/main/java/com/java/academy/gym/model/UserMessage.java

```diff
@@ -0,0 +1,73 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "userMessages")
+public class UserMessage extends BaseEntity{
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "locale_id", nullable = false)
+    private Locale locale;
+
+    @NotNull
+    @Size(max = 100)
```

Comment:
Message can be longer than 100. Minimum is 255(in my opinion it could be ~500)

Reply:
This is only message key, for message text I didn't set any max size.

Reply:
Ok, but still, key can be longer than 100.

Reply:
Okk